staticcheck.conf

```diff
@@ -0,0 +1 @@
+checks = ["inherit", "-ST1000", "-ST1005", "-ST1003"]
```

Comment:
Please can you add a comment here explaining what this does and what the rules are for.